verify-service/src/main/java/io/inji/verify/controller/VPResultController.java

```diff
@@ -35,7 +36,7 @@ public VPResultController(VerifiablePresentationRequestService verifiablePresent
     }
 
     @GetMapping(path = "/vp-result/{transactionId}")
-    public ResponseEntity<Object> getVPResult(@PathVariable String transactionId, HttpServletRequest request) throws CredentialStatusCheckException {
+    public ResponseEntity<Object> getVPResult(@PathVariable String transactionId, HttpServletRequest request) {
```

Comment:
Removing throws for CredentialStatusCheckException was already fixed with 0.16.0 LSH comments, rt? was this missed in develop? any other fixes that were missed?